tests/Resources/app.js

```diff
@@ -0,0 +1,235 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* global Ti, global */
+/* eslint no-unused-expressions: "off", no-global-assign: "off", no-native-reassign: "off" */
+'use strict';
+var utilities,
+	win,
+	$results = [],
+	failed = false,
+	should;
+
+require('./ti-mocha');
+// I *think* we need to load mocha first before utilities...
+utilities = require('./utilities/utilities');
+should = require('./utilities/assertions');
+win = Ti.UI.createWindow({
+	backgroundColor: 'yellow'
+});
+win.open();
+
+// Must test global is available in first app.js explicitly!
+// (since app.js is treated slightly differently than required files on at least Android)
+describe('global', function () {
+	it('should be available as \'global\'', function () {
+		should(global).be.ok;
+	});
+});
+
+// ============================================================================
+// Add the tests here using "require"
+// ES6 syntax/compatability tests
+require('./es6.arrows.test');
+require('./es6.default.args.test');
+require('./es6.rest.args.test');
+require('./es6.spread.args.test');
+require('./es6.string.interpolation.test');
+require('./es6.class.test');
+require('./es6.import.test');
+// Titanium APIs
+require('./ti.accelerometer.test');
+require('./ti.analytics.test');
+require('./ti.api.test');
+require('./ti.android.httpresponsecache.test');
```

Comment:
Once I merge to the full test suite, I'd modify the app.js like this, but for PRs the easier way to avoid it is to simply ensure your additional tests are named `ti.whatever.addontest.js` and they'll get picked up automatically. The `addontest.js` suffix is what gets it picked up.

Reply:
Remove app.js
Probably someone should update [README](https://github.com/appcelerator/titanium_mobile#unit-tests)